server/server.js

```diff
@@ -1,12 +1,29 @@
 const express = require('express');
 const bodyParser = require('body-parser');
+const path = require('path');
+const Log = require('log');
+const db = require('../db/index');
+
+const log = new Log('info');
 
 const app = express();
 
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
+app.use(express.static(path.join(__dirname, '../client/dist/')));
```

Comment:
Do you have an specific reason use the path library here?
> app.use(express.static('client/dist'));

this worked for me.